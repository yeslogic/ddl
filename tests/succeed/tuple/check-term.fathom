let Triple: Type = (U16, U16, U16);
let Point: Type = {x: U16, y: U16, z: U16};
let Any: Type = {A: Type, a: A};

let triple: Triple = (0, 1, 2);
let point: Point = (0, 1, 2);
let any: Any = (Bool, false);

()
