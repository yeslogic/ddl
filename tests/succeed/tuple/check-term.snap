stdout = '''
let Triple = (U16, U16, U16);
let Point = { x : U16, y : U16, z : U16 };
let Any = { A : Type, a : A };
let triple = (0, 1, 2);
let point = { x = 0, y = 1, z = 2 };
let any = { A = Bool, a = false };
() : ()
'''
stderr = ''
